packages/mwp-tracking-plugin/src/click.test.js

```diff
@@ -0,0 +1,99 @@
+import Hapi from 'hapi';
+import { plugin as loggerPlugin } from 'mwp-logger-plugin';
+import { plugin, CLICK_PLUGIN_NAME, onPreHandlerExtension } from './click';
+
+const mockClickReader = require('./util/clickReader');
+
+jest.mock('./util/clickReader', () => jest.fn());
+const h = { continue: Symbol('continue') };
+
+describe('onPreHandlerExtension', () => {
+	// try {
+	// 	const pluginSettings =
+	// 		request.route.settings.plugins[CLICK_PLUGIN_NAME] || {};
+	// 	if (pluginSettings.click && pluginSettings.click(request)) {
+	// 		return clickReader(request, h);
+	// 	}
+	// } catch (err) {
+	// 	request.server.app.logger.error({ err, context: request, ...request.raw });
+	// }
```

Comment:
Does this comment need to be here?

Reply:
ah, no, was using it as a reference to set up tests. will clean up!